scripts/common/preflights.sh

```diff
@@ -657,3 +658,18 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# bail_when_requires_minio_with_openebs will bail if OpenEBS and Minio not when it is required
+function bail_when_requires_minio_with_openebs() {
+    if [ -z "$MINIO_VERSION" ] && [ -n "$OPENEBS_VERSION" ]; then
+        if [ -n "$KOTSADM_VERSION" ] && [[ -z "$KOTSADM_DISABLE_S3" || "$KOTSADM_DISABLE_S3" != "1" ]];then
```

Comment:
`[ -z "$KOTSADM_DISABLE_S3" ]` != "1". this is redundant.

```suggestion
        if [ -n "$KOTSADM_VERSION" ] && [ "$KOTSADM_DISABLE_S3" != "1" ]; then
```